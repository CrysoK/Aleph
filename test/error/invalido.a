// Programa inválido (debe dar error)
¿ 3 // invalid token
+ // + inesperado
1_2 // _ inesperado
¡ // invalid token
! // ! inesperado
let a
b // no declarado
let a // re-declaración
println(1); println(2) println(3) println(4) // id inesperado
println(1) println(2) println(3); println(4) // id inesperado
