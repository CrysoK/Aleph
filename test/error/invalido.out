ERROR [Sintaxis] (test/error/invalido.a:1:1): Uno o más caracteres inválidos (no ASCII)
ERROR [Sintaxis] (test/error/invalido.a:2:1): Inesperado: '+'. Se esperaba: fin de línea, ';'.
ERROR [Sintaxis] (test/error/invalido.a:3:2): Inesperado: identificador. Se esperaba: fin de línea, ';'.
ERROR [Sintaxis] (test/error/invalido.a:4:1): Uno o más caracteres inválidos (no ASCII)
ERROR [Sintaxis] (test/error/invalido.a:5:1): Inesperado: '!'. Se esperaba: fin de línea, ';'.
ERROR [Símbolo] (test/error/invalido.a:7:3): Símbolo 'b' no declarado
ERROR [Sintaxis] (test/error/invalido.a:8:7): Re-declaración del símbolo 'a'
1
2
ERROR [Sintaxis] (test/error/invalido.a:9:24): Inesperado: identificador. Se esperaba: fin de línea, ';'.
1
ERROR [Sintaxis] (test/error/invalido.a:10:12): Inesperado: identificador. Se esperaba: fin de línea, ';'.
4
AVISO: 9 errores encontrados
